experimental/traffic-portal/src/app/core/servers/update-status/update-status.component.html

```diff
@@ -17,7 +17,7 @@ <h2 mat-dialog-title>Change status for {{serverName}}</h2>
 		<mat-form-field appearance="fill" *ngIf="true">
 			<mat-label>New Status</mat-label>
 			<select name="status" [(ngModel)]="status" required matNativeControl>
-				<option *ngFor="let status of statuses" [value]="status" [disabled]="status.id === currentStatus">{{status.name}}</option>
+				<option *ngFor="let status of statuses" [value]="status.name" [disabled]="status.id === currentStatus">{{status.name}}</option>
```

Comment:
were these changes to the update-status component necessary? I don't necessarily disagree that they don't simplify it, since apparently the name is the only information we need, but it clutters the PR to do a refactor in the midst of such a large, unrelated change.